Assets/Scripts/UI/WarningText.cs

```diff
@@ -0,0 +1,63 @@
+using UnityEngine;
+using UnityEngine.UI;
+
+public class WarningText : MonoBehaviour
+{
+    private enum alphaValue
+    {
+        SHRINKING,
+        GROWING,
+    }
+
+    private alphaValue currentAlphaValue;
+    [SerializeField] private float CommentminAlpha;
+    [SerializeField] private float CommentmaxAlpha;
+    [SerializeField] private float CommentCurrentAlpha;
+    [SerializeField] private bool flashText;
+
+    [SerializeField] private Text MyText;
```

Comment:
```suggestion
    [SerializeField] private TMPText text;
```
Zou ten eerst Text Mesh Pro gebruiken want het zit in het project. Ten tweede ik zal altijd "my" vermijden om varibels, want het is niet nettjes en Berend heeft dat ook een keer gezegd.